assets/src/tinymce/web-stories.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { render } from '@wordpress/element';
+import { dispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import '../../../node_modules/@wordpress/components/build-style/style.css';
+import name from './store/name';
+import WebStoryMCEStore from './store';
+import WebStoriesModal from './containers/Modal';
+import { webStoriesData } from './utils/globals';
+import { setFields } from './utils/';
+
+const { _ } = window;
+
+/**
+ * Add button to tinyMCE editor.
+ */
+(function () {
+  // eslint-disable-next-line no-prototype-builtins
+  if (!_.hasOwnProperty('pluck')) {
+    _.pluck = _.map;
+  }
```

Comment:
Where is `pluck` used? Why is this needed?

Reply:
@swissspidy `pluck` is used by core TinyMCE editor internally. If `pluck` is not present, there will be JS error related to functions' non-existence.

Reply:
Why is it our job to ensure `pluck` is present? Are we doing something that causes it to not be present?

Reply:
@swissspidy We are inserting the content in the editor dynamically [here](https://github.com/rtCamp/web-stories-wp/blob/feature/tinymce-button/assets/src/tinymce/components/Modal.js#L132) with the help of `insertContent` method.

This is basically the bug in WordPress, where WP is using `_pluck` in its implementation, but have updated the underscore library for which `_pluck` method is not available. This is the reason we have to apply this patch.

Reply:
Can you point me to where WP is using `pluck`? Is there a bug ticket in WordPress Trac or on GitHub about it?

And with updating underscore library I assume you mean WordPress is nowadays using Lodash instead of Underscore?

At very least, we should add an inline comment in the code, explaining what it does and why it's needed.

Reply:
Sure.

1. You can see that WP is using `_.pluck` method [here](https://github.com/WordPress/WordPress/blob/master/wp-includes/js/mce-view.js#L145).

2. Sorry, it was my mistake. WP is using lodash and lodash has removed `_pluck` method already. (Underscore also do not have _pluck method)

3. Screenshot of the error.
![pluck-error](https://user-images.githubusercontent.com/1442637/105984958-dba5de00-60c0-11eb-9d4a-f8a760cc5c44.png)

4. I did not find the relevant trac ticket for this issue. I will try to find it one more time.

5. Added the documentation for this patch.

Reply:
thank you